N/nativefiledialog_extended/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "nativefiledialog_extended"
+version = v"1.1.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/btzy/nativefiledialog-extended.git", "5786fabceeaee4d892f3c7a16b243796244cdddc")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/nativefiledialog-extended
+NFD_PORTAL=OFF
+NFD_APPEND_EXTENSION=OFF
+if [[ "${target}" == *-linux-* ]] ; then
+    NFD_PORTAL=ON
+elif [[ "${target}" == *-freebsd-* ]] ; then
+    NFD_APPEND_EXTENSION=ON
+fi
+cmake -B build \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_CXX_STANDARD=11 \
+    -DCMAKE_C_FLAGS="-Wall -Wextra -pedantic" \
+    -DCMAKE_CXX_FLAGS="-Wall -Wextra -pedantic" \
+    -DBUILD_SHARED_LIBS=ON \
+    -DNFD_PORTAL=${NFD_PORTAL} \
+    -DNFD_APPEND_EXTENSION=${NFD_APPEND_EXTENSION} \
+    .
+cmake --build build --parallel ${nproc}
+cmake --install build
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter!(p -> arch(p) != "armv6l", supported_platforms(; experimental=true))
```

Comment:
Also, do you need to filter out armv6l at all?

Reply:
Done too.

Reply:
> Also, do you need to filter out armv6l at all?

`Dbus`, and `Gtk3` does not mention `armv6l`. I don't know if it will work, but I put it to test the build.

Reply:
Ah, our dbus build is _old_.

Reply:
It was a failure for `armv6l` [here](https://buildkite.com/julialang/yggdrasil/builds/11098#01900be3-b49b-4c1b-8da0-846e8cf2eb6b).

Reply:
I'll try building against `SDL2` in those platforms, to see if it compiles.